src/nvim/ex_cmds.c

```diff
@@ -5908,27 +5856,24 @@ static buf_T *show_sub(exarg_T *eap, pos_T old_cusr, PreviewLines *preview_lines
   // Width of the "| lnum|..." column which displays the line numbers.
   linenr_T highest_num_line = 0;
   int col_width = 0;
+  handle_T preview_bufnr = get_preview_bufnr();
+  long preview_ns = get_preview_ns();
+  // Use preview window only when inccommand=split and range is not just the current line
+  bool preview = (*p_icm != 'n') && (eap->line1 != old_cusr.lnum || eap->line2 != old_cusr.lnum);
```

Comment:
"preview window" is `:help preview-window`. Try to avoid that, it's confusing. If we are really talking about the window, we could call it `cmdpreview window` in these comments.